src/server/db_slice.cc

```diff
@@ -804,7 +804,6 @@ void DbSlice::Release(IntentLock::Mode mode, const KeyLockArgs& lock_args) {
   if (lock_args.args.size() == 1) {
     string_view key = KeyLockArgs::GetLockKey(lock_args.args.front());
     ReleaseNormalized(mode, lock_args.db_index, key, 1);
-    uniq_keys_ = {key};
```

Comment:
Setting the uniq_keys_ was done in the PR #1611 to be able to check the keys after the command finished running, but nothing cleared it and this also leaded to bug in tests checks .Now we dont want to have any keys in this set after command finished running so I am just clearing the set on release